samples/modules/tensorflow/hello_world/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(tensorflow_hello_world)
+
+# Required for TensorFlow to compile properly
+set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fno-threadsafe-statics")
```

Comment:
Please make an entry in:
https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/compiler/compiler_flags_template.cmake

and minimum for GCC:
https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/compiler/gcc/compiler_flags.cmake
Clang supports the same flag and inherits its base from the gcc flags.

@evgeniy-paltsev does MWDT support the same flag ?

Reply:
Yep, ARC MWDT supports the `-fno-threadsafe-statics` flag